kevm-pyk/src/kevm_pyk/cli.py

```diff
@@ -480,7 +480,7 @@ class EVMChainOptions(Options):
     @staticmethod
     def default() -> dict[str, Any]:
         return {
-            'schedule': 'CANCUN',
+            'schedule': 'PRAGUE',
```

Comment:
Is this changing the default to `PRAGUE`? Should we wait for us to support it fully first? Or do we already support a lot of the functionality?

Reply:
Ah, yes! Good catch. I'll revert this change and set it as the default later.